### DISCUSSION TITLE
Looking for suggestions around adding custom node

### DISCUSSION THREAD

**Discussion by ben-smartbear** [markdoc/markdoc#363] [Q&A]
Hi! I'm trying to add a custom node matcher to parse and keep running into research dead ends.
My usecase is for something like:
```
$$item
some content
$$
```
Obviously the first thought is that a Tag would suit whatever purpose I'm trying to achieve, but in this case, the syntax requirement is very specific. Is such a custom Node possible with the options available?

**Comment by rpaul-stripe** [comment]
Markdoc doesn't support adding arbitrary new syntax. You can define a custom tag or you can use a fenced code block and add some custom behavior. Can you provide more details about what you are trying to accomplish?

**Reply by ben-smartbear** [reply]
We have markdown written using another tool which uses `$$ ... $$` as its plugin syntax. I have been able to use a fence to achieve what I want, but I was hoping there was a way for arbitrary syntax to avoid that. If there isn't, I can use the fence as I have already, or I can transform the markdown on the way in to migrate the other plugin syntax to a Markdoc compatible one.

**Reply by rpaul-stripe** [reply]
Yeah, a fence or conversion are probably your best options.
